tests/src/python/test_qgsgeometry.py

```diff
@@ -842,7 +842,7 @@ def testReferenceGeometry(self):
                                        "Perimeter {}: mismatch Expected:\n{}\nGot:\n{}\n".format(i + 1, exp, result))
 
     def testCollection(self):
-        g = QgsGeometry.fromWkt('MultiLineString()')
```

Comment:
I can't see a test for `MultiLineString()` in the newly added TestQgsGeometry::wktParser() test -- can you add one please?

Reply:
Done.